rllib/tuned_examples/simple_q/cartpole-simpleq-test.yaml

```diff
@@ -2,7 +2,7 @@ cartpole-simpleq-test:
     env: CartPole-v1
     run: SimpleQ
     stop:
-        episode_reward_mean: 150
-        timesteps_total: 50000
+        episode_reward_mean: 50.0
```

Comment:
is there a reason to be changing this and lowering the threshold so much? seems sus

Reply:
also why are you lowering the training timestep budget?

Reply:
Oh, good question. This is just a "test" config file. So the purpose of using this is really just to demo the CLI functionality. If you would like to really learn CartPole with SimpleQ, you'd be using the "normal" `cartpole-simpleq.yaml` file (w/o the "test").